src/Processors/Formats/Impl/PrettySpaceBlockOutputFormat.cpp

```diff
@@ -100,20 +76,31 @@ void PrettySpaceBlockOutputFormat::writeChunk(const Chunk & chunk, PortKind port
             const auto & type = *header.getByPosition(column).type;
             auto & cur_width = widths[column].empty() ? max_widths[column] : widths[column][row];
             writeValueWithPadding(
-                *columns[column], *serializations[column], row, cur_width, max_widths[column], cut_to_width, type.shouldAlignRightInPrettyFormats(), isNumber(type));
+                *columns[column],
+                *serializations[column],
+                row,
+                cur_width,
+                max_widths[column],
+                cut_to_width ? max_widths[column] : 0,
```

Comment:
Maybe let's define the 'cut_to_width' as it was previously? Because now the definition is a bit distributed, and it's harder to read the code.